src/sentry/api/serializers/rest_framework/rule.py

```diff
@@ -67,8 +67,12 @@ class RuleSerializer(serializers.Serializer):
     actionMatch = serializers.ChoiceField(
         choices=(("all", "all"), ("any", "any"), ("none", "none"))
     )
+    filterMatch = serializers.ChoiceField(
+        choices=(("all", "all"), ("any", "any"), ("none", "none")), required=False
+    )
```

Comment:
Do we need custom validation making this required if any `filters` are passed?

Reply:
good call 👍